ldes-fragmentisers/ldes-fragmentisers-timebased-hierarchical/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/timebasedhierarchical/model/FragmentationTimestamp.java

```diff
@@ -0,0 +1,83 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fragmentisers.timebasedhierarchical.model;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentisers.timebasedhierarchical.constants.TimeBasedConstants;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.fragmentisers.timebasedhierarchical.constants.TimeBasedConstants.*;
+
+public class FragmentationTimestamp {
```

Comment:
I feel like this class makes everything more complex than it should be. Using DateTimeFormatter this could be a lot simpler. Let's discuss this on gather